src/dxtbx/imageset.py

```diff
@@ -486,7 +493,7 @@ def _create_imageset(filelist, check_headers):
 
         # Get the template format
         if "#" in template:
-            filenames = sorted(_expand_template(template, indices))
+            filenames = _expand_template(template, indices)
```

Comment:
since `_expand_template` now adds sorting functionality could we find a better name? `expand_template_to_sorted_filenames`?